great_expectations/experimental/metric_repository/data_store.py

```diff
@@ -1,15 +1,15 @@
 import abc
 from typing import Generic, TypeVar
 
-from great_expectations.data_context import AbstractDataContext
+from great_expectations.data_context import CloudDataContext
 
 T = TypeVar("T")
 
 
 class DataStore(abc.ABC, Generic[T]):
     """Abstract base class for all DataStore implementations."""
 
-    def __init__(self, context: AbstractDataContext):
+    def __init__(self, context: CloudDataContext):
```

Comment:
This change was necessary since `ge_cloud_config` does not exist on `AbstractDataContext` and is used in `_build_url()`